contracts/libraries/Lockers.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.20;
+
+/// @notice This is a temporary library that allows us to use transient storage (tstore/tload)
+/// for the lockers array and nonzero delta count.
+library Lockers {
+    // The starting slot for an array of lockers, stored transiently.
+    uint256 constant LOCKERS_SLOT = uint256(keccak256("Lockers"));
```

Comment:
nitttt: the best-practice is to subtract 1 from this number, so that the preimage isnt known
see EIP1967!